### PR TITLE
fix: update premining interface

### PR DESCRIPTION
**Pull request by keeplearning-near**

### CODE REVIEW

File: src/libs/pool-data-provider/hooks/use-incentives-data-context.tsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ import {
 export interface IncentivesContext {
   reserveIncentives: ReserveIncentiveDict;
   userIncentives: UserIncentiveDict;
+  usersIncentives: UserIncentiveDict[];
```
Comment (linguists) [inline src/libs/pool-data-provider/hooks/use-incentives-data-context.tsx:39]: why `users`? the data is still for a single user

File: src/ui-config/networks.ts [hunk 2]
```diff
@@ -194,20 +194,24 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
...
     cachingServerUrl: 'https://cache-api-aurora.pret.app/graphql',
     cachingWSServerUrl: 'wss://cache-api-aurora.pret.app/graphql',
     baseUniswapAdapter: '0x0',
     baseAsset: 'ETH',
-    baseAssetWrappedAddress: '0xB8AD40DCfc163694F981D2e12704C0a7af9387DF',
+    baseAssetWrappedAddress: '0x88aabf01c7faab947d1570cd993bed8acebf54b2',
     // incentives hardcoded information
     rewardTokenSymbol: 'WNEAR',
-    rewardTokenAddress: '0x729dfF256Ba68502a17C1443B6F49c6B5300E95d',
+    rewardTokenAddress: '0x1a55e008e86ac4c170499e8e8af5d60565e4e453',
```
Comment (linguists) [inline src/ui-config/networks.ts:214]: do we need to configure multiple rewards tokens？

File: src/libs/pool-data-provider/hooks/use-incentives-data-context.tsx (hunk 1 above)
Comment (keeplearning-near) [reply src/libs/pool-data-provider/hooks/use-incentives-data-context.tsx]: resolved

File: src/ui-config/networks.ts (hunk 2 above)
Comment (keeplearning-near) [reply src/ui-config/networks.ts]: resolved.

**Comment by linguists** [comment]
The pre-mining UI is already implemented in #25

**Pull request closed without merging**